.github/workflows/release.yaml

```diff
@@ -82,11 +82,15 @@ jobs:
 
       - name: Build egctl multiarch binaries
         run: |
-          make build-multiarch BINS="egctl"
-          tar -zcvf egctl_${{ env.release_tag }}_linux_amd64.tar.gz bin/linux/amd64/
-          tar -zcvf egctl_${{ env.release_tag }}_linux_arm64.tar.gz bin/linux/arm64/
-          tar -zcvf egctl_${{ env.release_tag }}_darwin_amd64.tar.gz bin/darwin/amd64/
-          tar -zcvf egctl_${{ env.release_tag }}_darwin_arm64.tar.gz bin/darwin/arm64/
+          make build-multiarch
```

Comment:
can we add new make target to simply this?

Reply:
For the tar cmds ?